app/shared/components/Wallet/Status/Actions.js

```diff
@@ -0,0 +1,110 @@
+// @flow
+import React, { Component } from 'react';
+import { Header, Grid, Loader, Segment, Visibility } from 'semantic-ui-react';
+import { translate } from 'react-i18next';
+
+import ActionsTable from './Actions/Table';
+
+type Props = {
+  actions: {
+    getAccountActions: () => void
+  },
+  actionObjects: {},
+  t: () => void,
+  validate: {}
+};
+
+class Actions extends Component<Props> {
+  props: Props;
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      amount: 20
+    };
+  }
+
+  componentDidMount() {
+    this.tick();
+    this.interval = setInterval(this.tick.bind(this), 15000);
```

Comment:
The auto updating of this list isn't that easy - you can't just reload the last request with an "amount". If you were on page 10 with 10 on each page, it'd reload 100 actions on every request. 

The history of actions itself should be generated and then appended to, as opposed to reloading every 15 seconds.

Reply:
Yeah I think I'll get everything that was added since the lastTransaction as opposed to everything.

Reply:
I left this here, because we need to check for new actions, but that request will only check for new data and won't touch the list at all if there are none.